### PR TITLE
en: update doc for auto failover

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added, or deleted? (Required)

update doc for auto failover

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1594
- Other reference link(s):

### CODE REVIEW

File: en/use-auto-failover.md [hunk 1]
```diff
@@ -89,17 +89,29 @@ When the failed Pod in the cluster recovers, TiDB Operator **DOES NOT** remove t
...
+    {{< copyable "shell-regular" >}}
 
-```shell
-kubectl patch tc -n ${namespace} ${cluster_name} --type merge -p '{"spec":{"tikv":{"recoverFailover": true}}}'
-```
+    ```shell
+    kubectl patch tc -n ${namespace} ${cluster_name} --type merge -p '{"spec":{"tikv":{"recoverFailover": true}}}'
+    ```
+
+    Every time after the cluster recovers from failover, TiDB Operator automatically scale in the newly created Pods.
```
Comment (qiancai) [inline en/use-auto-failover.md:102]:
```suggestion
    Every time after the cluster recovers from failover, TiDB Operator automatically scales in the newly created Pods.
```

File: en/use-auto-failover.md [hunk 2]
```diff
@@ -116,17 +128,29 @@ When the failed Pod in the cluster recovers, TiDB Operator **DOES NOT** remove t
...
+    {{< copyable "shell-regular" >}}
 
-```shell
-kubectl patch tc -n ${namespace} ${cluster_name} --type merge -p '{"spec":{"tiflash":{"recoverFailover": true}}}'
-```
+    ```shell
+    kubectl patch tc -n ${namespace} ${cluster_name} --type merge -p '{"spec":{"tiflash":{"recoverFailover": true}}}'
+    ```
+
+    Every time after the cluster recovers from failover, TiDB Operator automatically scale in the newly created Pods.
```
Comment (qiancai) [inline en/use-auto-failover.md:141]:
```suggestion
    Every time after the cluster recovers from failover, TiDB Operator automatically scales in the newly created Pods.
```

**Review by qiancai — COMMENTED** [review]
@ran-huang two minor comments. PTAL

**Review by qiancai — APPROVED** [review]

**Review by better0332 — APPROVED** [review]

**Pull request merged**